packages/firestore/src/api/credentials.ts

```diff
@@ -26,7 +26,7 @@ import { Code, FirestoreError } from '../util/error';
 
 export interface FirstPartyCredentialsSettings {
   type: 'gapi';
-  client: Gapi;
+  client: unknown;
```

Comment:
Following up on this change-- might intro issues if there was any usage of client.x since Gapi was previously an any